doc/quickstart/install.rst

```diff
@@ -14,15 +14,15 @@ Once you have installed conda, you can install ESMValCore by running:
 
 .. code-block:: bash
 
-    conda install -c esmvalgroup -c conda-forge esmvalcore
+    conda install -c conda-forge esmvalcore
 
 It is also possible to create a new
 `Conda environment <https://docs.conda.io/projects/conda/en/latest/user-guide/getting-started.html#managing-environments>`_
 and install ESMValCore into it with a single command:
 
 .. code-block:: bash
 
-    conda create --name esmvalcore -c esmvalgroup -c conda-forge esmvalcore
+    conda create -y --name esmvaltool -c conda-forge 'python=3.9'
```

Comment:
```suggestion
    conda create --name esmvalcore -c conda-forge esmvalcore 'python=3.9'
```
or the new environment will only contain python and not esmvalcore. Do you think we should change the name of the environment too?

Reply:
Also, I think it is important that the instructions are the same as what we do on CircleCI, but this is different

Reply:
man, I keep eating important syntax today :rofl: Hmmm I think it's fine `esmvaltool` - one can start getting confused if we change names, I already had a case of a student using two different environments each for Core and Tool

Reply:
> Also, I think it is important that the instructions are the same as what we do on CircleCI, but this is different

This is what we do on Circle CI, the problem is that we don't test other bits of the instructions, like the direct `conda install -c conda-forge esmvalcore` call (but I think we test that in GA)

Reply:
and of course I changed the env name :man_facepalming: - I need a beer :grin:

Reply:
oh no I didn't, but I still need a beer :beer: